Makefile.include

```diff
@@ -20,6 +20,10 @@ RIOTPKG ?= $(RIOTBASE)/pkg
 RIOTPROJECT ?= $(shell git rev-parse --show-toplevel 2>/dev/null || pwd)
 RIOTPROJECT := $(abspath $(RIOTPROJECT))
 
+# add git tools directory to PATH so git can pick it up
+PATH:=$(PATH):$(RIOTBASE)/dist/tools/git
+export PATH
```

Comment:
I'm not sure I would like it if a build script would modify my PATH variable. Maybe you can store the old value and restore it after the build?